sqlacodegen/codegen.py

```diff
@@ -19,6 +19,12 @@
 from sqlalchemy.types import Boolean, String
 from sqlalchemy.util import OrderedDict
 
+# Support CIText and CIText[] in PostgreSQL via sqlalchemy-citext
+try:
+    from citext import CIText  # noqa: F401
+except (NameError, ImportError):
```

Comment:
This should be moved to after the Geoalchemy2 import.

Under what circumstances can this raise NameError?

Reply:
Oops, that was supposed to be `ModuleNotFoundError` not `NameError`. Though I realize now that's overly specific and will fail on Python2 anyway as it doesn't have `ModuleNotFoundError`- which is a subclass of `ImportError`. Will remove that and commit, nice catch

Reply:
(fixing the order as well)